src/activity_handlers.cpp

```diff
@@ -1176,6 +1171,7 @@ void activity_handlers::pickaxe_finish( player_activity *act, player *p )
         p->mod_thirst( 5 );
         p->mod_fatigue( 10 );
     }
+    p->add_msg_if_player( m_good, _( "You finished digging." ) );
```

Comment:
`You finish digging` (present tense) would be the correct tense here, for this and the Burrowing message.

Note, by the way, that there are 2 other instances of the incorrect tense (vs. 8 instances of the correct tense) in this file that you could move as well if so inclined:

https://github.com/CleverRaven/Cataclysm-DDA/blob/249d508c770587ed8a3b7890b6af529e5dc05bea/src/activity_handlers.cpp#L2089

https://github.com/CleverRaven/Cataclysm-DDA/blob/249d508c770587ed8a3b7890b6af529e5dc05bea/src/activity_handlers.cpp#L2129